### ISSUE TITLE
MAJOR ISSUES

### ISSUE THREAD

**Issue by rifsxd** [statulr/aurc#9]
Bro why did you remove the issue where i talked about copyright infringement and carry on not fixing it, you know i can dmca this repo cuz its my code, i ask you politely to do the changes within 24 hours.

**Comment by statulr** [comment]
I made the changes, not only that you blocked me when I asked why you blocked me, you chose to leave this project and block, and I replied to the issue about that, I asked many ppl in the dev community and they said that I should delete the issue, and DMCA isn't possible as the copyright license clearly states anyone can use this, it is open source, and the code has changed so so much since you left. So if you want to play that type of game, I am shocked. Why did you leave the project and say you wanted do you your own thing and that I could keep it and continue it if you didnt want me to? That was your choice, alright. I owned the org, the site, most of the code, and you left me stranded, that ruined 90% of my motivation as you blocked me without reason and etc. So be reasonable at least.

**Comment by statulr** [comment]
Check recent commits from 20 hours ago :)

**Comment by statulr** [comment]
Your name is included if you like, but please understand what you did, and that's why this happened.

**Comment by rifsxd** [comment]
And no the code is still 90% mine the foundation the core and front end and everything, and thats okay but you cant remove my name from copyright

**Comment by rifsxd** [comment]
About blocking idk what happened my wife might have done that accidentally and unblocked soon after while i was sleeping last night

**Comment by rifsxd** [comment]
You should let others know in the readme is this project is based on the original aurc from my repo

**Comment by statulr** [comment]
> About blocking idk what happened my wife might have done that accidentally and unblocked soon after while i was sleeping last night

really? that's your excuse, twice? so your wife coincidentally blocked someone right after you said i can have it and that your leaving

**Comment by statulr** [comment]
> I left aurc org to do my separate thing that later on i abandoned to move to Ubuntu for work reasons no hard feelings in all this i simply ask to e give proper attribution

I did.

**Comment by statulr** [comment]
> Bro i never blocked you but i unfriended a lot of people and yes you can do the changes on top my code but that doesnt mean you can remove my copyright tho that is infringement

Yes you did... Lying really? I attributed you, that's the end of it. I don't realize what you intend to gain, I already credited you

**Comment by rifsxd** [comment]
i never blocked you neither did my wife after asking all i did was do an unfriend streak and about attribution you contrubuted to aurc not me to you thats the issue

**Comment by statulr** [comment]
> i never blocked you neither did my wife after asking all i did was do an unfriend streak and about attribution you contrubuted to aurc not me to you thats the issue

You literally blocked me as it didn't let me friend you either, you just changed your story also check the bottom

**Comment by statulr** [comment]
> and btw aurc has a lot of memory leaks

Make a new issue if you believe so

**Comment by statulr** [comment]
I will delete this soon, solely because I have been recommended by others to do so, as it is not an issue relating to the actual code, this has been resolved.
